### PR TITLE
Example and smoke test

### PR DESCRIPTION
**Pull request by patflynn**
Signed-off-by: Patrick Flynn <patrick@chainguard.dev>

### CODE REVIEW

File: examples/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ARG BASE=distroless.dev/alpine-base
+
+FROM $BASE
+CMD ["echo", "Hello Distroless!"]
```
Comment (amouat) [inline examples/Dockerfile:4]: Does this work without the exec syntax i.e. `CMD echo "Hello distroless"`?

**Review by amouat — APPROVED** [review]
Example is fine, but I'm not sure it's really needed, you could just do:

docker run $IMAGE_NAME echo "hello distroless"

**Comment by patflynn** [comment]
🤷  that's a fair point.  I guess the end-users will likely to include it in a dockerfile so perhaps it helps. Or we could use a hello-world java app instead to make it more interesting?

File: examples/Dockerfile (hunk 1 above)
Comment (patflynn) [reply examples/Dockerfile:4]: sure. I'll switch to it if you want to keep this example.

**Comment by amouat** [comment]
So, I would:
 - keep the smoke test as simple and fast as we can. For that reason, I'd drop the Docker build line
 - add whatever you think is useful to examples. I probably wouldn't give an example of compiling something, as that's better done by the language images.

**Comment by patflynn** [comment]
I added a non static go app example and moved to the simpler smoke test.

I'm using the non distroless go builder LMKWYT

**Comment by patflynn** [comment]
I added a non static go app example and moved to the simpler smoke test.

I'm using the non distroless go builder LMKWYT

**Comment by amouat** [comment]
Yeah, we could go through all our examples and change to the distroless builders! But this is fine for the moment I think.

**Comment by amouat** [comment]
Nice one @patflynn - thanks for this :)

**Pull request merged**